tests/conftest.py

```diff
@@ -388,6 +388,7 @@ def run_remote_command(instances, command):
                 "mender-monitor",
                 "mender-gateway",
                 "mender-client",
+                "mender",
```

Comment:
This is really old, client 2.2 and older :laughing: 

(but I understand you added it to print errors from compat test - :ok_hand: )

Reply:
:older_adult: